assets/src/blocks/amp-story-text/edit.js

```diff
@@ -116,26 +123,59 @@ function TextBlock( props ) {
 					/>
 				</PanelColorSettings>
 			</InspectorControls>
-			<RichText
-				identifier="content"
-				wrapperClassName="wp-block-amp-story-text"
-				tagName="p"
-				value={ content }
-				onChange={ ( value ) => setAttributes( { content: value } ) }
-				style={ {
-					backgroundColor: backgroundColor.color,
-					color: textColor.color,
-					fontSize: fontSize.size ? fontSize.size + 'px' : undefined,
+			<ResizableBox
+				className={ classnames(
+					'amp-story-text__resize-container',
+					{ 'is-selected': isSelected }
+				) }
+				size={ {
+					height,
+					width,
+				} }
+				minHeight={ minTextHeight }
+				minWidth={ minTextWidth }
+				enable={ {
+					top: true,
+					right: true,
+					bottom: true,
+					left: true,
+					topRight: false,
+					bottomRight: false,
+					bottomLeft: false,
+					topLeft: false,
```

Comment:
Not sure how many "dots" we should create for the Text blocks for resizing. Perhaps just each side? Or perhaps just the corners?

Reply:
Let's test with each side (Gutenberg default I think?) and then test with corners if it doesn't feel right.

Reply:
Sounds good!